userspace/falco/webserver.cpp

```diff
@@ -34,6 +34,23 @@ k8s_audit_handler::~k8s_audit_handler()
 {
 }
 
+k8s_healthz_handler::k8s_healthz_handler()
+{
+}
+
+k8s_healthz_handler::~k8s_healthz_handler()
+{
+}
+
+bool k8s_healthz_handler::handleGet(CivetServer *server, struct mg_connection *conn)
+{
+	std::string status_body = "{\"status\": \"ok\"}";
```

Comment:
```suggestion
	const std::string status_body = "{\"status\": \"ok\"}";
```

Reply:
thanks @fntlnz for your review, did the change and also update https://github.com/falcosecurity/falco/pull/1546/files#diff-759cf16077c5a03cbf4e9b6dd8af34208252447adb0f3f894e5aadb6f218335dR168 to match your feedback here 😄 

PTAL